hs/test-examples/features/lib_default_app.rsh

```diff
@@ -0,0 +1,12 @@
+'reach 0.1';
+
+// Test assertions for libaries
+export const [ isOutcome, B_WINS, DRAW, A_WINS ] = makeEnum(3);
+
+export const winner = (handA, handB) =>
```

Comment:
Can you go one step more and put this in `examples` and have a test case that actually uses these exports?